packages/main/src/AvatarGroup.js

```diff
@@ -69,6 +70,19 @@ const metadata = {
 			defaultValue: AvatarGroupType.Group,
 		},
 
+		/**
+	 	 * Defines the total avatar count of the <code>AvatarGroup</code>.
```

Comment:
Defines the total number of avatars that logically belong to this <code>ui5-avatar-group</code>.
If not given, the total number of slotted <code>ui5-avatar</code> elements will be used.

<b>Note:</b> Use this property if there is a large number of items, represented by the <code>ui5-avatar-group</code>, and you'd rather not load them all in DOM due to performance reasons. In such a scenario, just slot a few <code>ui5-avatar</code> instances and set this property to reflect their true total number.